README.md

```diff
@@ -15,42 +15,71 @@ In contrast, our philosophy is rooted in simplicity:
 2. Prioritize clarity
 3. Embrace simplicity in design
 
-We believe this approach is the best way to build a client that is both fast and resilient. By adhering to these principles, we will be able to iterate fast and explore next-generation features early, either from the Ethereum roadmap or from innovations from the L2s.
+We believe this approach is the best way to build a client that is both fast and resilient. By adhering to these principles, we will be able to iterate fast and explore next-generation features early, either from the Ethereum roadmap or from innovations from the L2s. 
+
+## Quick Start (localnet)
+![Demo][./demo.png]
+You'll need docker, docker compose and rust 1.81.
+### Mac
+```shell
+brew install kurtosis-tech/tap/kurtosis-cli && make
+```
+### Linux
+[Install kurtosis][https://docs.kurtosis.com/install/#ii-install-the-cli] and run:
+```shell
+make
+```
+
+This will:
+- Start [ethereum package][https://github.com/ethpandaops/ethereum-package], a private testnet on which multiple ethereum clients can interact.
+- Start our execution client, and make it interact with the others.
 
-## Usage
+## Dev Setup
 
 ### Build
```

Comment:
Mixed feelings about this, I would recomment asdf instead since we don't necesarily support the latest rust version (although we probably should).